src/MIDebugEngine/AD7.Impl/AD7Engine.cs

```diff
@@ -572,13 +572,20 @@ public int Continue(IDebugThread2 pThread)
             // VS Code currently isn't providing a thread Id in certain cases. Work around this by handling null values.
             AD7Thread thread = pThread as AD7Thread;
 
-            if (_pollThread.IsPollThread())
+            try
             {
-                _debuggedProcess.Continue(thread?.GetDebuggedThread());
+                if (_pollThread.IsPollThread())
+                {
+                    _debuggedProcess.Continue(thread?.GetDebuggedThread());
+                }
+                else
+                {
+                    _pollThread.RunOperation(() => _debuggedProcess.Continue(thread?.GetDebuggedThread()));
+                }
             }
-            else
+            catch (InvalidCoreDumpOperationException)
             {
-                _pollThread.RunOperation(() => _debuggedProcess.Continue(thread?.GetDebuggedThread()));
+                return AD7_HRESULT.E_CRASHDUMP_UNSUPPORTED;
```

Comment:
Looking at the vscode side, it doesn't look like we always check the return codes to all the ad7 calls, such as Continue and Bind.